pilot/pkg/model/telemetry.go

```diff
@@ -404,23 +406,27 @@ func (t *Telemetries) telemetryFilters(proxy *Proxy, class networking.ListenerCl
 // mergeLogs returns the set of providers for the given logging configuration.
 // This currently is just the names of providers as there is no access logging configuration, but
 // in the future it will likely be extended
-func mergeLogs(logs []*tpb.AccessLogging, mesh *meshconfig.MeshConfig) sets.Set {
+func mergeLogs(logs []*tpb.AccessLogging, mesh *meshconfig.MeshConfig) (sets.Set, *tpb.AccessLogging_Filter) {
 	providers := sets.NewSet()
 
 	if len(logs) == 0 {
 		for _, dp := range mesh.GetDefaultProviders().GetAccessLogging() {
 			// Insert the default provider.
 			providers.Insert(dp)
 		}
-		return providers
+		return providers, nil
 	}
-
+	var alsFilter *tpb.AccessLogging_Filter
```

Comment:
very nit - `als` is for access log *service*, but this applies to all access logs?